plugins/com.google.cloud.tools.eclipse.appengine.localserver/plugin.xml

```diff
@@ -163,4 +163,27 @@
       </processFactory>
    </extension>
 
+   <extension
+         point="org.eclipse.core.runtime.preferences">
+      <initializer
+            class="com.google.cloud.tools.eclipse.appengine.localserver.PreferencesInitializer">
+      </initializer>
+   </extension>
+   <extension
+         point="org.eclipse.ui.preferencePages">
+      <page
+            category="com.google.cloud.tools.eclipse.preferences.main"
+            class="com.google.cloud.tools.eclipse.preferences.areas.AreaBasedPreferencePage"
+            id="com.google.cloud.tools.eclipse.preferences.appengine.localserver"
+            name="Run/Debug">
```

Comment:
This pref page is directly under "Google", so "App Engine Run/Debug" seems more nature. Or, perhaps just "App Engine"?